site/app/templates/admin/UploadConfigForm.twig

```diff
@@ -25,12 +25,41 @@
 
     <br><br>
 
-    <form action="{{ upload_url }}" method="POST" enctype="multipart/form-data">
+    <form id = "configUpload" action="{{ upload_url }}" method="POST" enctype="multipart/form-data">
         <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
-        <label>Upload Config: <input type="file" name="config_upload" /><br /></label>
+        <label>Upload Config: <input type="file" id="configFile" name="config_upload" /><br /></label>
         <input type="submit" value="Upload" /><br />
         <button type='reset'>Cancel Upload</button>
     </form>
+
+    <script>
+    let unUploadedFile = false;
+    document.getElementById("configUpload").addEventListener("change", function(event) {
```

Comment:
You can use jQuery to eliminate a lot of boilerplate syntax. `$('#config-upload').on(...` or similar syntax is much nicer to read.